test/server_test.py

```diff
@@ -222,7 +241,7 @@ def started_imageio(tmpdir, drop_privileges="true"):
 
         # Wait until server is listening - at this point it already dropped
         # privileges.
-        if drop_privileges:
+        if drop_privileges and ticket is None:
```

Comment:
why dropping privileges is related to not having a ticket? These features should not have any dependencies.

Reply:
I was not interested in running the non-existing ticket check if I am testing the initial ticket, and drop_privileges is always evaluated as true since it is a string.

Reply:
ok, this is a side effect of testing for missing ticket as a way to test that the server is responding.

We better change the test to test something else instead, since new we have new use case of starting a server with a ticket.

We can test instead `OPTIONS /images/*` or `GET /info` which does not depend on the control service or on having a ticket.